avocado/core/job.py

```diff
@@ -373,6 +373,8 @@ def _log_avocado_datadir():
         job_log.info('')
 
     def _log_mux_tree(self, mux):
+        if not mux.variants:
```

Comment:
This is an intermittent change, but yes, I plan to add documentation fixes. With so many changes I am actually considering, whether it would be OK to fix the documentation in the last big change and I'm not talking about improving the documentation, but rather complete rework (keeping only parts).

Before the documentation was covered in the `Mux` section, after this PR I'd split it into `Multiplexer` section to talk about params, basic tree structure and the variants API. Then I'd add `Yaml to mux` section, which would talk about variants generation, the extended tree structure and the algorithm to produce the variants used in this plugin.

What do you think, would that be acceptable? I don't think I can easily rewrite the current documentation as it combines booth of them and does not cover the multiplexer API at all.